Modify the instructions of the ingress configuration (#3301)
knative/docs: docs/install/knative-with-operators.md

   metadata:
     name: knative-serving
     namespace: knative-serving
   EOF
   ```

>    There is no need to configure the ingress class to use the gloo.

Reviewer: ```suggestion
   You do not need to configure the ingress class to use Gloo.
```

>    There is no need to configure the ingress class to use the gloo.

Reviewer 2: Maybe this is set by `glooctl`?

>    There is no need to configure the ingress class to use the gloo.

Author: we do not need it for gloo.
we only ship contour, kourier and istio.
for gloo, ambassador, kong, people can just configure and use. no need to set anything with knative.
